apps/sms/views/conversation/js/conversation.js

```diff
@@ -2480,26 +2483,23 @@ var ConversationView = {
     // force a number
     var id = +messageDOM.dataset.messageId;
     var iccId = messageDOM.dataset.iccId;
-
-    var request = MessageManager.retrieveMMS(id);
```

Comment:
Just throwing some thoughts I had here, but I now think it's better like this.
I was wondering if we could do the call in MessageManager instead. Maybe having 2 different "MessageManager" implementations, switchable using a setting. OR MessageManager itself could decide to use the client or mozMobileMessage directly.

But now I think this be the role of MessagingClient, deciding to either use a worker or to load the worker script inside the window for a fast path.